Etch.OrchardCore.CivicCookieControl.csproj

```diff
@@ -21,17 +21,17 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Etch.OrchardCore.Fields" Version="1.6.0" />
-    <PackageReference Include="OrchardCore.Admin" Version="1.7.2" />
-    <PackageReference Include="OrchardCore.ContentFields" Version="1.7.2" />
-    <PackageReference Include="OrchardCore.ContentManagement" Version="1.7.2" />
-    <PackageReference Include="OrchardCore.ContentManagement.Display" Version="1.7.2" />
-    <PackageReference Include="OrchardCore.DisplayManagement" Version="1.7.2" />
-    <PackageReference Include="OrchardCore.Flows" Version="1.7.2" />
-    <PackageReference Include="OrchardCore.Navigation" Version="1.7.2" />
-    <PackageReference Include="OrchardCore.ResourceManagement" Version="1.7.2" />
-    <PackageReference Include="OrchardCore.Module.Targets" Version="1.7.2" />
-    <PackageReference Include="OrchardCore.Title" Version="1.7.2" />
+    <PackageReference Include="Etch.OrchardCore.Fields" Version="1.7.0" />
```

Comment:
@peterkeating Should I have upgraded the fields package?

Reply:
Yes! :)